utils/ercot/fully-describe-data-product.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
these scripts are nifty! 

just to confirm, they were manually run and then the results copied and pasted into `ercot.py`? just trying to think through in the future when someone want to implement a new report.

is the idea that they would run these scripts again to add to `ercot.py`? if so, we should leave a comment in that file about these so they are discoverable by future contributors